.travis.yml

```diff
@@ -7,5 +7,8 @@ services:
 
 script:
   - docker build -t android-sdk ./android-sdk
+  - docker build -t android-sdk-firebase-test-lab ./android-sdk/firebase-test-lab
   - docker images
   - cmd=$(cat ./version_inspector.sh) && docker run -it --rm android-sdk bash -c "$cmd"
+  - cmd=$(cat ./android-sdk/firebase-test-lab/version_inspector.sh) && docker run -it --rm android-sdk-firebase-test-lab bash -c "$cmd"
```

Comment:
Do we want firebase-test-lab inside android-sdk, right? @thyrlian 
Because if I remove "./android-sdk" this command will fail :sweat_smile:

Reply:
Correct, sorry it was my fault :P